src/test/java/core/basesyntax/model/FruitTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTest {
```

Comment:
You don't need to test classes that have no functionality. It's just a model.

Reply:
If I would made those tests after morning lecture I would add Fruit and TransactionDto together with main to pom.xml. But the pluses are I know that all get methods and equals works correctly.